backend/dissemination/sql/fac-snapshot-db/post/143_api_v2_0_0_views.sql

```diff
@@ -0,0 +1,112 @@
+---------------------------------------
+-- additional_eins
+---------------------------------------
+CREATE VIEW api_v2_0_0.additional_eins AS
+    SELECT * FROM public_data_v1_0_0.additional_eins ae
+    ORDER BY ae.id;
+
+---------------------------------------
+-- additional_ueis
+---------------------------------------
+create view api_v2_0_0.additional_ueis AS
+    SELECT * FROM public_data_v1_0_0.additional_ueis au
+    ORDER BY au.id;
+
+---------------------------------------
+-- corrective_action_plan
+---------------------------------------
+CREATE VIEW api_v2_0_0.corrective_action_plans AS
+    SELECT * FROM public_data_v1_0_0.corrective_action_plans cap
+    ORDER BY cap.id;
+
+---------------------------------------
+-- finding
+---------------------------------------
+CREATE VIEW api_v2_0_0.findings as
+    SELECT * FROM public_data_v1_0_0.findings f
+    ORDER BY f.id;
+
+---------------------------------------
+-- finding_text
+---------------------------------------
+CREATE VIEW api_v2_0_0.findings_text AS
+    SELECT * FROM public_data_v1_0_0.findings_text ft
+    ORDER BY ft.id;
+
+---------------------------------------
+-- federal award
+---------------------------------------
+CREATE VIEW api_v2_0_0.federal_awards AS
+    SELECT * FROM public_data_v1_0_0.federal_awards fa
+    ORDER BY fa.id;
+
+---------------------------------------
+-- general
+---------------------------------------
+CREATE VIEW api_v2_0_0.general AS
+    SELECT * FROM public_data_v1_0_0.general gen
+    ORDER BY gen.id;
+
+---------------------------------------
+-- notes_to_sefa
+---------------------------------------
+CREATE VIEW api_v2_0_0.notes_to_sefa AS
+    SELECT * FROM public_data_v1_0_0.notes_to_sefa nts
+    ORDER BY nts.id;
+
+---------------------------------------
+-- passthrough
+---------------------------------------
+CREATE VIEW api_v2_0_0.passthrough AS
+    SELECT * FROM public_data_v1_0_0.passthrough p
+    ORDER BY p.id;
+
+---------------------------------------
+-- auditor (secondary auditor)
+---------------------------------------
+CREATE VIEW api_v2_0_0.secondary_auditors AS
+    SELECT * FROM public_data_v1_0_0.secondary_auditors sa
+    ORDER BY sa.id;
+
+---------------------------------------
+-- combined
+---------------------------------------
+CREATE VIEW api_v2_0_0.combined AS
+    SELECT * FROM public_data_v1_0_0.combined comb
+    ;
+
```

Comment:
census_gsa_crosswalk view is missing here.  This needs to be added.

Reply:
Apologies; I still didn't get to that. 

You could add it and do a PR against the codebase. However, it also wants to be added in the `.sql` in the... `fac-snapshot-db/post` folder.

Actually, I'd be happy to pair (Monday?) for 30m to help add this in. It would be faster, I suspect, for both of us.